src/interface/IEspressoNitroTEEVerifier.sol

```diff
@@ -1,23 +1,56 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+import "./IEspressoTEEVerifier.sol";
+
 interface IEspressoNitroTEEVerifier {
     // This error is thrown when the PCR0 values don't match
     error InvalidAWSEnclaveHash();
 
-    event AWSEnclaveHashSet(bytes32 enclaveHash, bool valid);
-    event AWSSignerRegistered(address signer, bytes32 enclaveHash);
-    event DeletedAWSRegisteredSigner(address signer);
+    event AWSEnclaveHashSet(
+        bytes32 enclaveHash, bool valid, IEspressoTEEVerifier.ServiceType service
+    );
+    event AWSServiceRegistered(
+        address signer, bytes32 enclaveHash, IEspressoTEEVerifier.ServiceType service
+    );
+    event DeletedAWSRegisteredService(address signer, IEspressoTEEVerifier.ServiceType service);
+
+    /*
+    * @notice This function is for checking the registration status of AWS Nitro TEE Caff Nodes and is a helper function for the EspressoTEEVerifier
+    */
+    function registeredCaffNode(address signer) external view returns (bool);
 
-    function registeredSigners(address signer) external view returns (bool);
+    /*
+    * @notice This function is for checking the registration status of AWS Nitro TEE Batch Posters and is a helper function for the EspressoTEEVerifier
+    */
+    function registeredBatchPoster(address signer) external view returns (bool);
     function registeredEnclaveHash(bytes32 enclaveHash) external view returns (bool);
 
-    function registerSigner(bytes calldata attestation, bytes calldata data) external;
+    /*
+    * @notice This function is for registering AWS Nitro TEE Caff Nodes and is a helper function for the EspressoTEEVerifier
+    */
+    function registerCaffNode(bytes calldata attestation, bytes calldata data) external;
+
+    /*
+    * @notice This function is for registering AWS Nitro Batch Posters and is a helper function for the EspressoTEEVerifier
+    */
+    function registerBatchPoster(bytes calldata verificationData, bytes calldata data) external;
```

Comment:
Is it specified already as to what will be the `verificationData` and the `data`?